client/src/app/components/page/list/list.component.html

```diff
@@ -4,19 +4,83 @@ <h4>Filter your tasks</h4>
     </select2>
 
 </div>
-<div class="container">
-    <div class="row">
-        <div *ngFor="let task of filteredTasks; let i = index" class="col-lg-3 col-md-4 col-sm-6 col-xs-12">
-            <a [routerLink]="['/tasks', task._id]">
-                <div class="card mt-3 cardCol">
-                    <h5 class="card-title">{{task.title}}</h5>
-                </div>
+
+<!--
+
+    <div class="container mt-3">
+        <ul class="list-group">
+            <li class="list-group-item">
+                <span>
+                    <b>Title</b>
+                </span>
+                <span class="float-right mr-3">
+                    <b>Tags</b>
+                </span>
+            </li>
+            <li *ngFor="let task of filteredTasks" class="list-group-item listElement">
+                <a [routerLink]="['/tasks', task._id]">
+                    {{task.title}}
+                    <div *ngFor="let tag of task.tags"
+                        class="float-right mr-3 rounded-lg bg-secondary text-white tagContainer">
+                        {{tag}}
+                    </div>
+                </a>
+            </li>
+        </ul>
+    
+        <div class="addTaskButton float-right">
+            <a [routerLink]="['/tasks/newTask']" class="">
+                <button class="btn btn-outline-success my-2 p-2 my-sm-0">Add new task</button>
             </a>
         </div>
     </div>
-    <div class="addTaskButton">
-        <a [routerLink]="['/tasks/newTask']">
-            <button class="btn btn-outline-success my-2 my-sm-0">Add task</button>
+-->
+
+<div class="container my-3">
+    <table class="table">
+        <thead>
+            <tr>
+                <th>#</th>
+                <th>
+                    Title
+                </th>
+                <th>
+                    <div class="float-right mr-3">
+                        Tags
+                    </div>
+                </th>
+                <th>
+                    <div class="float-right mr-3">
+                        Created at
+                    </div>
+                </th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr *ngFor="let task of filteredTasks; let i = index">
+                <th>{{i+1}}</th>
+                <td>
+                    <a [routerLink]="['/tasks', task._id]">
+                        {{task.title}}
+                    </a>
+                </td>
+                <td>
+                    <div *ngFor="let tag of task.tags"
+                        class="float-right mr-3 rounded-lg bg-secondary text-white tagContainer">
+                        {{tag}}
+                    </div>
+                </td>
+                <td>
+                    <div class="float-right mr-3">
+                        {{getDateString(task.createdAt)}}
```

Comment:
You can use angular date pipe here https://angular.io/api/common/DatePipe

Reply:
Changed to date piping